src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -476,4 +478,83 @@ public void Stop()
 
 		NativeMethods.Stop(IsAudio ? _htmlAudio.HtmlId : _htmlVideo.HtmlId);
 	}
+
+	public double _playbackRate;
+	public double PlaybackRate
+	{
+		get => _playbackRate;
+		set
+		{
+			_playbackRate = value;
+			if (IsVideo)
```

Comment:
Instead of having all these "IsVideo"/"IsAudio" logic, can we have a simple property that wraps this eg
ActiveElement => IsVideo? _htmlVideo:
                                    IsAudio? _htmlVideo: 
                                             default;